README.md

```diff
@@ -1 +1,37 @@
-# angular-what-is-a-directive-readme
\ No newline at end of file
+# What is a Directive?
+
+## Overview
+
+You might not have noticed, but we've used directives already in our previous labs. Directives are essential to learning Angular, so hold in tight!
+
+## Objectives
+
+- Describe what Directives are
+
+## Directives, deep down
+
+Directives can be compared to components. They are essentially "markers" on a HTML element that tell Angular to attach a specific behaviour to the HTML element - either something small like a click event, or actually completely transforming the DOM node to display a list of data.
+
+Directives can be either an actual HTML element or an attribute on a HTML element.
+
+```html
+<my-directive></my-directive>
+
+<!-- these are the same -->
+
+<div my-directive></div>
+```
+
+However, not all directives can be used in both ways - some are restricted to being just an attribute or just an element.
+
+## Types of directives
+
+There are two types of directives - event and behavioural. Event directives handle all of the events on a HTML element we might need - for example, `click`, `mouseover`, `keydown`, etc.
```

Comment:
I would mention that the event directives control what actions happen when certain events are trigged. Maybe you could compare this to event handlers when using jQuery.